### ISSUE TITLE
SurrealDB?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/technology-stack#104]
Rust Powered Database SurrealDB (It's Pretty Ambitious): https://youtu.be/DPQbuW9dQ7w

https://surrealdb.com/
https://github.com/surrealdb/surrealdb

this is _definitely_ one to watch. 🤩

Shared by @pratik-IDWT 🤓

**Comment by nelsonic** [comment]
Getting started with SurrealDB: https://youtu.be/D41jb4DDIdA

**Comment by nelsonic** [comment]
Asked if anyone has experience on Elixir Forum: https://elixirforum.com/t/has-anyone-used-surrealdb-with-elixir-phoenix/50577

**Comment by nelsonic** [comment]
SurrealDB - Rust Embedded Database - Quick Tutorial: https://youtu.be/iOyvum0D3LM
[![image](https://user-images.githubusercontent.com/194400/204655554-9bde8166-6d13-49df-9f38-081f6b1ce793.png)](https://youtu.be/iOyvum0D3LM)

**Comment by nelsonic** [comment]
https://surrealdb.com/
![image](https://user-images.githubusercontent.com/194400/206945643-dc54fc7c-17f9-47a1-81d4-94671f3e7991.png)

**Comment by nelsonic** [comment]
https://github.com/surrealdb/surrealdb
![image](https://user-images.githubusercontent.com/194400/206945775-3167b32d-33e8-4f64-b903-d7684e7d25c5.png)

**Comment by nelsonic** [comment]
Hosting Surreal DB, written in Rust, in under 3 minutes: https://youtu.be/VoRoeL1tal4
[![image](https://user-images.githubusercontent.com/194400/206947854-30b9b446-6fc4-4e83-b359-b1cddc18e0f0.png)](https://youtu.be/VoRoeL1tal4)

**Comment by nelsonic** [comment]
https://medium.com/featurepreneur/surreal-db-the-ultimate-cloud-database-78f0c2dfde1d
![image](https://user-images.githubusercontent.com/194400/206948419-8270c4e1-5b13-4938-b725-05e7e2c319c7.png)

**Comment by nelsonic** [comment]
The _official_ Deploying SurrealDB page:
https://www.surrealdb.dev/docs/guides/deploying-surrealdb
![image](https://user-images.githubusercontent.com/194400/207183556-3ad02c04-6c17-4f6a-86a7-4cb0da018166.png)

`Lorem ipsum` indeed. 🙄

**Comment by nelsonic** [comment]
Beyond SurrealDB: https://youtu.be/LCAIkx1p1k0

**Comment by nelsonic** [comment]
Sadly, at the time of writing the _official_ `Flutter` client for `SurrealDB` is "coming soon" ... 
https://surrealdb.com/docs/integration/libraries/flutter
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/194400/215184549-da4a25bf-88d3-46af-bbf5-f73db27ac5b8.png">

**Comment by pratik-IDWT** [comment]
@nelsonic  - I saw the surrealDB roadmap and might be of interest for remainder of 2023....

<img width="1009" alt="image" src="https://github.com/dwyl/technology-stack/assets/112123058/6786371f-d5c8-413e-997a-f3afd8298674">

**Comment by nelsonic** [comment]
Yeah, this is good news. I'm tracking it closely. Will seriously consider using it in an upcoming project. 💭 
Hope you're all well @pratik-IDWT ❤️
